src/views/Project.vue

```diff
@@ -510,10 +510,17 @@ export default Vue.extend({
     const handleMouseMove = (e: MouseEvent) => {
       window.getSelection().removeAllRanges();
       const width = window.innerWidth - e.clientX;
-      if (width >= core.app.view.project.state.tool['minWidth']) {
-        core.app.view.project.state.tool['width'] = width;
-        window.dispatchEvent(new Event('resize'));
+      if (
+        width ===
+        core.app.view.project.state.tool['minWidth']
+      ) {
```

Comment:
ESLint suggests
```suggestion
      if (width === core.app.view.project.state.tool['minWidth']) {
```
Can you confirm that?